e2e/README.md

```diff
@@ -0,0 +1,31 @@
+# E2E tests
+
+
+## Directory structure
+
+- `test-app-v10`: example app using Angular v10 with basic setup with `jest` and `jest-preset-angular`.
+- `test-app-v9`: example app using Angular v9 with basic setup with `jest` and `jest-preset-angular`.
+
+
+## Running tests
+
+You run the tests on example apps with `yarn test:e2e` (or `npm run test:e2e)`. In `scripts/lib/paths.js`,
+`projectsToRun` must be modified if a new project is added to `e2e`.
```

Comment:
could be determined automatically using `glob` package on `e2e/` directory. However, I'd much prefer using Jest's "projects" to facilitate e2e testing, which would get rid of custom sync tests running script

Reply:
will do in a separate PR to setup yarn workspace and use jest "projects"